src/tools/singlejar/combiners_test.cc

```diff
@@ -39,7 +39,9 @@ class CombinersTest : public ::testing::Test {
     ASSERT_EQ(0, system("zip -qm combiners.zip tag1.xml tag2.xml"));
```

Comment:
Here and everywhere where we call `system`: [MSDN says](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/system-wsystem?view=vs-2017#remarks) you must `fflush` / `_flushall` first.

Reply:
I think the flush is only needed so that `stdout` will display on console without weird surprises. There are quite a lot of `system` calls in the tests.

If you think flush is really necessary, I will add a wrapper function that wraps `system` and flush in the wrapper function (otherwise we will have `#ifdef _WIN32 _flushall()` everywhere).

Reply:
It seems like situations where flushing is really required before calling `system` is quite rare (when `system`'s command uses the same unclosed input/output files from the main program, which does not apply to singlejar tests).

I agree with the OP of the thread below that it is weird for Microsoft's implementation to pose such a requirement when other platforms do not require this.

https://groups.google.com/forum/#!topic/microsoft.public.vc.language/urQTYtpxSyM

Reply:
Makes sense, thanks for the explanation!